hbase_storage/src/main/java/com/splicemachine/access/hbase/HBasePartitionAdmin.java

```diff
@@ -810,4 +810,14 @@ public int getTableCount() throws IOException{
             throw e;
         }
     }
+
+    @Override
+    public void createSITable(String tableName) throws StandardException{
+        try {
+            HBaseConnectionFactory instance = HBaseConnectionFactory.getInstance(SIDriver.driver().getConfiguration());
+            instance.createSITable(HBaseConfiguration.CONGLOMERATE_SI_TABLE_NAME);
```

Comment:
It looks like the parameter to instance.createSITable should be tableName.

Reply:
Good catch!